schema/schema.py

```diff
@@ -22,20 +30,22 @@ def format_param(param: click.core.Option) -> Dict[str, Union[Optional[str], Lis
                        a list of those allowed options
     * default - The default option for that parameter
     """
-    formatted_param: Dict[str, Union[Optional[str], List[str]]] = {"name": param.name, "help": param.help}
+    formatted_param: Dict[str, Any] = {"title": param.name, "description": param.help}
 
     # NOTE: Params do not have explicit "string" type; either "text" or "path".
     #       All choice options are from a set of strings.
-    if param.type.name in ["text", "path", "choice"]:
+    if param.type.name in ["text", "path", "choice", "filename", "directory"]:
         formatted_param["type"] = "string"
+    elif param.type.name == "LIST":
+        formatted_param["type"] = "array"
     else:
         formatted_param["type"] = param.type.name
 
     if param.default:
-        formatted_param["default"] = str(param.default)
+        formatted_param["default"] = list(param.default) if isinstance(param.default, tuple) else param.default
```

Comment:
Just a question, what are some instances of a default being a tuple of something?

Reply:
The only one I was able to find was `sam sync --capabilities`, which has a default value of `('CAPABILITY_NAMED_IAM', 'CAPABILITY_AUTO_EXPAND')` (as a tuple). Even though it's just the one instance, JSON schema doesn't accept tuples, so I cast them to list, which will have the same effect in the schema.